components/layout/index.coffee

```diff
@@ -38,6 +38,26 @@ $ ->
 
   $('body').on 'click', '.js-language-toggle', toggleLanguageOnClick
 
-
+  $('body').on 'click', '.header-section-nav a', (e)->
+    section = $(e.currentTarget).data 'section'
+    $('html,body').animate { scrollTop: $("##{section}").offset().top - 140 }, 1000
+
+  # headerImageLink = "./images/Banner_"
+  incrementer = 1
+
+  changeHeaderBackground = ->
+    # $('.header').animate { 'background-image': 'url(' + headerImageLink + (incrementer++) + '.jpg)' }, 1000
+    # $('.header').css('background-image', 'url(' + headerImageLink + (incrementer++) + '.jpg)')
+    # $('.header').css('background-image', "url(./images/Banner_#{incrementer++}.jpg)")
+    $('.header').css {
+    "background-image": "url(./images/Banner_#{incrementer++}.jpg)",
+    "-webkit-transition": "all 1s ease-in",
```

Comment:
I would add these styles on the class itself. Since these transitions aren't changing, there is no reason to re-set them every time the background toggles.